.github/workflows/check-3rd-party.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+FILE=$1
+status=0
+
+while read -r line; do
+  src=`echo $line | sed -E "s/(.*)uses://g" | xargs`
+  author=`echo $src | awk -F/ '{print $1}'`
```

Comment:
I'd check that this is actually github. That is, check URL prefixes (e.g. `github.com/actions`) instead of users.

Reply:
GitHub Actions are not included by URL, but by username/repo.
I don’t think anyone can publish an action using the “actions” username, but if you really want to make it stricter, we could create a whitelist. I don’t think that this is worth the effort though.

Reply:
Ah, I see.